src/ui/simple_ui/ask_outfile.rs

```diff
@@ -101,11 +101,19 @@ impl fmt::Display for ListOption {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
             ListOption::Device(dev) => {
-                write!(
-                    f,
-                    "{} | {} - {} ({}, removable: {})",
-                    dev.name, dev.model, dev.size, dev.target_type, dev.removable
-                )?;
+                if dev.target_type == device::Type::Disk {
+                    write!(
+                        f,
+                        "{} | {} - {} ({}, removable: {})",
+                        dev.name, dev.model, dev.size, dev.target_type, dev.removable
+                    )?;
+                } else {
+                    write!(
+                        f,
+                        "{} | {} - {} ({})",
+                        dev.name, dev.model, dev.size, dev.target_type
+                    )?;
+                }
```

Comment:
nit: i would prefer this as a `match` instead of `if`

Reply:
@ifd3f Done :)